librz/analysis/arch_platform.c

```diff
@@ -105,6 +105,18 @@ static bool sdb_load_arch_platform_by_path(RZ_NONNULL RzArchPlatformTarget *t, R
 	return result;
 }
 
+RZ_API const char *rz_arch_platform_add_comments_references(RzArchPlatformTarget *t, ut64 port) {
```

Comment:
```suggestion
RZ_API const char *rz_arch_platform_get_item(RzArchPlatformTarget *t, ut64 port) {
```
I think the name `add_comments_references` is not right. This function just returns the comment associated with a port.